extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -19,7 +19,7 @@
   --i-amphtml-interactive-img-option-gap: 1em !important;
   --i-amphtml-interactive-img-transform: scale(0.95) !important;
 
-  border-radius: 3.75em 3.75em 4.375em 4.375em !important;
+  border-radius: 3.75em 3.75em 50% 50% / 3.75em 3.75em 4.375em 4.375em !important;
```

Comment:
Nit: Would it work if it only has one value for horizontal and vertical radii if the bottom ones are larger than the img-option-container (just `3.75em 3.75em 4.5em 4.5em`)?

Reply:
I thought using this strategy would mess up the prompt section for img quiz/polls with 2 options, but I tested it and it works because there's a small region where the chip is straight along the sides.

I just pushed the changes.